webui/react/src/pages/ResourcePool/ResourcepoolDetail.tsx

```diff
@@ -277,7 +316,11 @@ const ResourcepoolDetailInner: React.FC = () => {
             size={ShirtSize.Large}
           />
         </Section>
-        {!!topologyAgentPool.length && poolname && <Topology nodes={topologyAgentPool} />}
+        {agents.length <= MAX_USABLE_NODES && totalSlots <= MAX_USABLE_SLOTS && (
```

Comment:
I'd probably share the check here w/ thee one that's deciding to load agents w/ slots so that they don't go out of sync

Reply:
will fix it

Reply:
fixed